tests/test_configdict.py

```diff
@@ -0,0 +1,31 @@
+""" Tests for plaster_pastedeploy.ConfigDict
+"""
+import copy                     # noqa: F401
+import plaster
+import pytest
+
+
+@pytest.fixture
+def loader():
+    from plaster_pastedeploy import Loader
+    uri = plaster.PlasterURL('pastedeploy+ini', 'development.ini')
+    return Loader(uri)
+
+
+@pytest.mark.parametrize('copy_method', [
+    "configdict.copy()",
```

Comment:
Make this a lambda that is called instead of something that is `eval`ed? Same thing with the next one? Accept a single argument?

Reply:
I had originally written it that way (passing functions rather than lambdas.)  The reason I switched to the eval was that it makes the diagnostic output from pytest more useful.  With the string-valued parameters, pytest names the generated tests something like `test_copy[configdict.copy()]` and `test_copy[copy.copy(configdict)]` rather than `test_copy[copyfunc-0]` and `test_copy[copyfunc-1]`.

If you care strongly, I'll happily change it, but that was my motivation for the `eval`.

Reply:
You can specify explicit ids for the parametrized tests if this is a concern:

https://hackebrot.github.io/pytest-tricks/param_id_func/

See under the header "explicit ids"

Reply:
I've just pushed a commit which eliminates the eval. See if you like it.